### ISSUE TITLE
import cPickle 在python3 环境有问题

### ISSUE THREAD

**Issue by alice-cool** [yanxinzju/CSS-VQA#14]
我改成了import _pickle as cPickle
然后在数据预处理时，报错
vals = list(map(float, vals[1:]))
![feb077d81bf6621670285c5cb2f554b](https://user-images.githubusercontent.com/16870890/187818574-8178e614-b997-44aa-bae0-74a8f6d80d02.png)
但是在运行main 时又出现新的编码问题 utf-8
![image](https://user-images.githubusercontent.com/16870890/187818664-90217fde-7e63-4c7b-bd4c-88b76ad226d8.png)
通过
fe=torch.load('data/rcnn_feature/'+str(img_id)+'.pth', encoding="ISO-8859-1")['image_feature']
classifier
nn.Dropout(dropout, inplace=False)
又跑通了

**Comment by pjy0422** [comment]
Thank you very much. I had the same error and thanks to you I solved it.
